types/atom-mocha-test-runner/tslint.json

```diff
@@ -0,0 +1,36 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        // Custom rules.
+        "class-name": true,
```

Comment:
There should be no other rules here, it should be only `"extends": "dtslint/dt.json"`, 
if you really want to disable some rule, you can disable them in code

Reply:
Is this actually written anywhere? These are extra enables, with zero disables.